### DISCUSSION TITLE
After approval

### DISCUSSION THREAD

**Discussion by Shadvaladi** [community/community#17722] [GitHub Education] [answered]
Sir I have submitted application for education pack which has been Approved so how will I give them my address and what happens after approval.
Please reply

**Comment by mlarasusan** [comment, marked as answer]
Hello [Shadvaladi](https://github.com/Shadvaladi), thanks for reaching out. If you have been approved already, you should receive a welcome email with all the information [☺️](https://emojipedia.org/smiling-face/)
